scripts/sidedock.py

```diff
@@ -12,7 +13,10 @@ def toggle_box():
     boxobj.show = not boxobj.show
     viewer.renderer.update()
 
+def slider_changed(value):
+    print(f"Slider value changed to: {value}")
 
 viewer.ui.sidedock.show = True
 viewer.ui.sidedock.add(Button(text="Toggle Box", action=toggle_box))
+viewer.ui.sidedock.add(Slider(title="test", min_val=0, max_val=10, action=slider_changed))
```

Comment:
For example I tried to add step=0.5 or step=2 here but it doesn't really do anything on my computer

Reply:
Had feeling this could be some issue with pyside itself. if this feature does not really work we can for now remove step...

Reply:
`self.slider.setSingleStep(self.step)` actual means the increment step when u click on the slider, now is working